node/node_test.go

```diff
@@ -164,6 +164,11 @@ func TestNodeStartMining(t *testing.T) {
 		assert.Error(t, err, "node is already mining")
 	})
 
+	t.Run("bootstrap miner does not set a storage fault slasher", func(t *testing.T) {
```

Comment:
This description is wrong. The test appears to be testing that "mining start initializes a storage fault slasher".